### DISCUSSION TITLE
Example with element-plus

### DISCUSSION THREAD

**Discussion by Oleksii14** [vitest-dev/vitest#285] [General]
Hello! I was trying to set up the vitest env for a big project with Element plus. I receive some errors. Could you add some examples?

**Comment by wangjunxian** [comment]
have you solved the problems when you use Element plus in vitest ?
I met many problems when I use vitest to do test.
like  [Failed to resolve component: el-input],
[formEl.validate is not a function].

**Reply by Oleksii14** [reply]
This discussion is very old and not actual, sorry 🙏🏻 
@wangjunxian
